README.md

```diff
@@ -8,11 +8,11 @@
 </div>
 
 <h1 align="center">
-Data and AI algorithms into production-ready web apps
+Build scalable data and AI apps, go beyond Streamlit 
```

Comment:
Not a fan of this. About the Streamlit comparison

Reply:
changing to other libraries